README.md

```diff
@@ -54,5 +55,9 @@ Just run `git-copy-history` again and it will add only the new commits.
 git-copy-history from ../local-repo
 git push
 ```
+## Update your profile settings
+
+If you have private repository and did not check this box: go [here](https://github.com/settings/profile) and check the box:
```

Comment:
Instead of pointing `here` I suggest using the exact place. Like `Profile Settings page`

Reply:
I mean like that:
<img src="https://user-images.githubusercontent.com/584632/62839812-e4221300-bc98-11e9-8455-59d346d84357.gif" width="741"/>

Reply:
Wrong thread :)